engines/wintermute/detection_tables.h

```diff
@@ -480,6 +480,64 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("reversion1", "Version 1.3.2369",
 		WME_ENTRY2s("xlanguage_pt.dcp", "886886b6b14aadac844078de856799a6", 10620797,
 					"data.dcp", "aecb5deeea7b0baa871fbd0cef35a648", 254219204), Common::PT_BRA, ADGF_UNSTABLE, LATEST_VERSION),
+	
+	// Reversion: The Escape (Steam release) (Spanish)
+	//
+	// The data for the Spanish version are all in data.dcp.
+	// Because we rely on WME_ENTRY2s to detect the languages,
+	// we have to detect data.dcp twice. For some reason, the
+	// Spanish version of the game only works if **no** xlanguage_*.dcp
+	// file is present in the game directory. If any xlanguage_*.dcp
+	// file exists in the game directory, the detection entry for the
+	// Spanish version will launch another language instead.
+	//
+	// I (rootfather) think that the problem is that there's no
+	// special xlanguage_*.dcp file for Spanish. The original game
+	// seems to switch between languages by simply copying the desired
+	// language file from /languages to the main folder. 
+	//
+	// To enable Spanish speech, remove xvoice_en.dcp from your game directory.
+	// To enable English speech, copy xvoice_en.dcp from the /languages directory
+	// to the game directory.
+	WME_WINENTRY("reversion1", "Steam",
+		WME_ENTRY2s("data.dcp", "5e4d40075f69fa7702530e38c349d2fd", 254293949,
+					"data.dcp", "5e4d40075f69fa7702530e38c349d2fd", 254293949), Common::ES_ESP, ADGF_UNSTABLE, LATEST_VERSION),
```

Comment:
What you need to use here is WME_ENTRY1s() macro:

   WME_ENTRY1s("data.dcp", "5e4d40075f69fa7702530e38c349d2fd", 254293949), 

Or you mean that when you're using it with one entry, the language is not detected correctly?

If so, then you're nicely exploiting the fact that AD makes preference by the number of matched files, which is 2 in this case.